mmv1/third_party/terraform/website/docs/r/compute_attached_disk.html.markdown

```diff
@@ -101,6 +101,18 @@ The following arguments are supported:
 	  "READ_ONLY"
 	  "READ_WRITE"
 
+* `interface` -
+  (Optional)
+  The disk interface used for attaching this disk. One of SCSI or NVME.
+
+	This field is only used for specific cases, please don't specify
+	this field without advice from Google. Not specifying the feild
+	will allow the the server to assign the correct interface.
+
+	Possible values:
```

Comment:
This is contained in the first line of this description, so we can skip one or the other